ui/frontend/webpack.config.js

```diff
@@ -158,7 +158,7 @@ module.exports = function(_, argv) {
       }),
       new MonacoWebpackPlugin({
         filename: `${filenameTemplate}.worker.js`,
-        languages: ['rust'],
+        languages: [],
```

Comment:
Why this change?

Reply:
We don't need their grammar because we use our own anyway. It should save some bits.